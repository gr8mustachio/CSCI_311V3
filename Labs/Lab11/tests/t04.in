8 9
0 3
0 4
2 6
3 2
4 2
5 2
5 7
1 5
7 6
dfs
