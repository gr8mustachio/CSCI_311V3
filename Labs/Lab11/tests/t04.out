Graph is:
0: 3 4 
1: 5 
2: 6 
3: 2 
4: 2 
5: 2 7 
6: 
7: 6 
Running DFS:
0 3 2 6 4 1 5 7 
